Projects/App/Sources/UI/Review/EvaluationViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EvaluationViewController.swift
+//  App
+//
+//  Created by 리아 on 2022/10/20.
+//  Copyright (c) 2022 zesty. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+import SnapKit
+
+final class EvaluationViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let cancelBag = Set<AnyCancellable>()
+    
+    private lazy var evaluationStackView = UIStackView()
+    private lazy var goodButton = EvaluationButton(type: .good)
+    private lazy var sosoButton = EvaluationButton(type: .soso)
+    private lazy var badButton = EvaluationButton(type: .bad)
+
+    // viewModel한테서 옵셔널로 받아올 거임
+    // 사용자가 뒤로 갈 수도 있으니까
+    private var evaluation: Evaluation!
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+    }
+    
+    // MARK: - Function
+    
+    @objc func goodButtonTouched() {
+        goodButton.isSelected.toggle()
+        sosoButton.isSelected = false
+        badButton.isSelected = false
+        
+        evaluation = .good
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+    @objc func sosoButtonTouched() {
+        sosoButton.isSelected.toggle()
+        badButton.isSelected = false
+        goodButton.isSelected = false
+        
+        evaluation = .soso
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+    @objc func badButtonTouched() {
+        badButton.isSelected.toggle()
+        goodButton.isSelected = false
+        sosoButton.isSelected = false
+        
+        evaluation = .bad
+        navigationController?.pushViewController(ReviewRegisterViewController(), animated: true)
+
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension EvaluationViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .systemBackground
+        
+        evaluationStackView.axis = .horizontal
+        evaluationStackView.spacing = 10
```

Comment:
p1;

<img width="553" alt="image" src="https://user-images.githubusercontent.com/75792767/197540441-d14e02cb-8e9d-4e49-8479-9a5bd3bb86b8.png">

여기 간격 15입니다!

```suggestion
        evaluationStackView.spacing = 15
```

Reply:
9a5214929a64a8203485870f4a8d812d2989f655 반영 완료~